tests/cannon/ctor1.dora

```diff
@@ -0,0 +1,12 @@
+//= output "xyz"
+
+@cannon class X(let msg: String) {
```

Comment:
Does this test even run the new delegation code? AFAIU there should only be a delegation-call when the class has a super class.

Reply:
You're correct, it did not. I added another test, which should rest delegation.